### ISSUE TITLE
Adding A "Send Message" Button 

### ISSUE THREAD

**Issue by BakeFlowers** [Chatterino/chatterino2#2279]
**What should be added?**
Please Add a "Send Message" Button like in the browser version of Twitch chat 

**Why should it be added?**
Tbh Im old and have multiple uses for this feature, since i grew up with it. 

But to completly honest, its mostly me being to lazy to press enter or hanging out on the couch only with the mouse.

**Comment by pawndini** [comment]
I agree 100%!  Can you please add an "enter" button that is clickable via a trackball.

Reason: I am a C5 quadriplegic so my fingers don't work and I have to use voice software to type. I can click my trackball with the back of my hand, but I have to bring my on-screen keyboard up, then click on the text field that I have just entered with my voice software, Then click the enter button on my keyboard, Then minimize my on-screen keyboard again.  I'm slow at using my mouse to begin with, but all these other steps I'm usually unable to post my message in a timely fashion. I also used to have this problem with the browser version of discord, but the PC has solved that.  Thanks for taking the time to read this :-)

**Comment by pajlada** [comment]
In v2.4.4, you will be able to enable a Send button. It will also be available in the Nightly build in about an hour after this message.

It looks like this:
![image](https://user-images.githubusercontent.com/962989/236676076-28b11bf4-8fc2-4c25-9c04-2846ec84251b.png)

It's disabled by default, but can be enabled with the "Show send message button" setting
![image](https://user-images.githubusercontent.com/962989/236676101-14e72d29-826c-453e-a65a-5fcd2837a8aa.png)

**Comment by pawndini** [comment]
Big thanks :-)
